cmd/machine-api-migration/main.go

```diff
@@ -216,29 +219,34 @@ func main() {
 		os.Exit(1)
 	}
 
-	machineSyncReconciler := machinesync.MachineSyncReconciler{
+	// ClusterOperator watches and keeps the cluster-api ClusterObject up to date.
+	if err := (&clusteroperator.ClusterOperatorController{
```

Comment:
Is this the same controller as the other manager? Is it safe to have it running twice?